rust/agama-server/src/cert.rs

```diff
@@ -0,0 +1,63 @@
+use openssl::error::ErrorStack;
+use openssl::pkey::{PKey, Private};
+use openssl::rsa::Rsa;
+use openssl::x509::extension::{KeyUsage, SubjectAlternativeName};
+use openssl::x509::{X509NameBuilder, X509};
+
+use openssl::asn1::Asn1Time;
+use openssl::bn::{BigNum, MsbOption};
+use openssl::hash::MessageDigest;
+use openssl::x509::extension::{BasicConstraints, SubjectKeyIdentifier};
```

Comment:
here it use two `use` sharing same `openssl::x509::extension::` which is quite confusing.